dev-tools/devmachine/devmachine.go

```diff
@@ -0,0 +1,115 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package devmachine
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	compute "google.golang.org/api/compute/v1"
+)
+
+func Run(instanceName string, imageName string, zone string) error {
+	ctx := context.Background()
+
+	projectID := "elastic-platform-ingest"
+	log.Println("Authenticating with GCP...")
+	computeService, err := compute.NewService(ctx)
+	if err != nil {
+		log.Fatalf("Failed to create compute service: %v. \nRun `gcloud auth login` to make it work", err)
+	}
+
+	imageProject := "elastic-images-prod"
+	machineType := fmt.Sprintf("zones/%s/machineTypes/n1-standard-4", zone)
+	sourceImage := fmt.Sprintf("projects/%s/global/images/%s", imageProject, imageName)
```

Comment:
All of these strings should be constants imo.

Reply:
I searched through the [code entries](https://github.com/search?q=zones%2F%25s%2FmachineTypes+language%3AGo&type=code&l=Go) and these strings are usually inlined. That's reasonable because it helps better understand the final string

Reply:
Moved the templates to constants